canbench-bin/src/fmt.rs

```diff
@@ -27,7 +27,7 @@ pub(crate) fn fmt_human_i64(value: i64) -> String {
     }
 }
 
-pub(crate) fn fmt_human_percent(value: f64) -> String {
+pub(crate) fn fmt_percent(value: f64) -> String {
```

Comment:
Why are we dropping "human"?

Reply:
because 'human' applies more to non-percentage values that can be huge and therefore may require unit suffixes, eg. K/M/B/T.
and we use those values in 2 forms: with and without units for table (easy to read) and CSV (sortable columns).
but percentages are only used in one form, therefore there's no need to have 'human' in the name.